include/qengine_cpu.hpp

```diff
@@ -41,7 +41,7 @@ class QEngineCPU : public QEngine, public ParallelFor {
         complex phaseFac = complex(-999.0, -999.0), bool doNorm = true, bool randomGlobalPhase = true,
         bool ignored = false);
     QEngineCPU(QEngineCPUPtr toCopy);
-    ~QEngineCPU() { delete[] stateVec; }
+    ~QEngineCPU() { free(stateVec); }
```

Comment:
Well derp. Create a FreeStateVec function for symmetry.

Reply:
Will do.